stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -121,6 +121,80 @@ export.hash_to_point.8
     dropw
 end
 
+# PUBLIC-KEY-HASH RECOVERY FROM SIGNATURE
+# =============================================================================================
+
+#!   Operand stack: []
+#!   Advice stack:  [tau0, tau1, h_0, ..., h_511, s2_0, ..., s2_511, pi_0, ..., pi_1022, ...]
+#! Outputs:
+#!   Operand stack: [PK_HASH]
+export.falcon_recover_pub_key.16
```

Comment:
Why do we allocate 16 locals for this procedure? It doesn't seem we actually use any of them.